package.json

```diff
@@ -44,10 +44,16 @@
     "monaco-editor": ">=0.30"
   },
   "devDependencies": {
+    "path-browserify": "^1.0.0",
+    "url": "^0.11.0",
     "@types/tailwindcss": "^3.0.0",
     "@typescript-eslint/eslint-plugin": "^5.0.0",
     "eslint": "^8.0.0",
     "eslint-config-remcohaszing": "^3.0.0",
-    "prettier": "^2.0.0"
+    "prettier": "^2.0.0",
+    "tailwindcss": "^3.0.0",
+    "tailwindcss-language-service": "0.0.23",
+    "postcss": "8.3.9",
+    "postcss-selector-parser": "6.0.2"
```

Comment:
those deps where moved to devDependencies to be able to stub the nodebuiltins - which is good for the dev-user but also a bit unfortunate, as they really increase the bundle size.
* "tailwindcss"
* "tailwindcss-language-service"
* "postcss"
* "postcss-selector-parser"

Reply:
wow es2015 is so 2015 ^^

Reply:
but that means we could build tailwindcss-language-service ourselves better?